packages/main/test/specs/MultiComboBox.spec.js

```diff
@@ -73,12 +73,12 @@ describe("MultiComboBox general interaction", () => {
 
```

Comment:
Let's continue to use the open since it's the public property and at the same time with will return the value of _open